### PR TITLE
Docs on data submission

### PR DESCRIPTION
**Pull request by kirstenwinther**

### CODE REVIEW

File: docs/source/tutorials/upload.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
+Submitting data
+---------------
+
+Please submit your electronic structures calculations for surface reactions! Data submissions will be part of the Surface Reactions app (essentially CatApp v2.0) and is open to all institutions. Furthermore, the atomic structures that are part of your reaction can be utilized for other apps.
+
+Your publication/dataset will be listed on the http://www.catalysis-hub.org/publications page, with a link to the publishers homepage (if already published). Your data will be easily accessible to other researchers, who will be able to browse through reaction energies and you atomic structures from your publication.
```
Comment (mhoffman) [inline docs/source/tutorials/upload.rst]: "and you atomic structures" -> "atomic structures"

**Comment by kirstenwinther** [comment]
@max - Please feel free to add to the documentation if something is missing.

File: docs/source/tutorials/upload.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+* SHERLOCK : /home/winther/data_catapp/
+* SHERLOCK2 : /home/users/winther/data_catapp/
+* SUNCAT : /nfs/slac/g/suncatfs/data_catapp/
+
+On SHERLOCK2 you can go to the directory and run::
+
+  . CATKITENV/bin/activate
+
+to use the local installation of CatKit. Alternatively you can install your own version of CatKit - see instructions below.
+
```
Comment (mhoffman) [inline docs/source/tutorials/upload.rst]: I would add comment like: 

You should see a `(CATKITENV) `at the beginning of your prompt now to indicate that all python script and libraries are first imported from the virtualenv. To return your shell to the previous state simply type `deactivate` or log out.

File: docs/source/tutorials/upload.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    "doi": "10.1002/cssc.201500322", 
+    "title": "The Challenge of Electrochemical Ammonia Synthesis: A New Perspective on the Role of Nitrogen Scaling Relations",
+    "journal": "ChemSusChem",
+    "authors": ["Montoya, Joseph H.", "Tsai, Charlie", "Vojvodic, Aleksandra", "Norskov, Jens K."],
+    "year": "2015",
+    "number": "13",
+    "tags": [],
+    "pages": "2140-2267"
+  }
+Note that authors should be a list, with names in the form "lastname, firstname M.".
```
Comment (mhoffman) [inline docs/source/tutorials/upload.rst]: Add: "Please go through the created folder and rename folders to make your data easier to localize later. For example a structure folder like Pt16_structure, could be changed to Pt16_fcc or Pt16_bcc respectively. Please do not use spaces in any folders or folders."

**Review by mhoffman — APPROVED** [review]
I approve after adopting the few minor changes indicated below :-).

**Pull request merged**